## cd-text-hldtst.dpatch by Kees Cook <kees@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add CD-TEXT support to "HL-DT-ST" "DVDRAM GSA-H42L" burner.

--- a/dao/CdrDriver.cc
+++ b/dao/CdrDriver.cc
@@ -177,6 +177,7 @@
 { "generic-mmc", "GENERIC", "CRD-BP1600P", 0, NULL },
 { "generic-mmc", "GENERIC", "CRD-R800S", 0, NULL },
 { "generic-mmc", "GENERIC", "CRD-RW2", 0, NULL },
+{ "generic-mmc", "HL-DT-ST", "DVDRAM GSA-H42L", OPT_MMC_CD_TEXT, NULL },
 { "generic-mmc", "HL-DT-ST", "RW/DVD GCC-4120B", OPT_MMC_CD_TEXT, NULL },
 { "generic-mmc", "HP", "9510i", OPT_MMC_CD_TEXT, NULL },
 { "generic-mmc", "HP", "CD-Writer+ 7570", OPT_MMC_CD_TEXT, NULL },
--- a/dao/cdrdao.drivers
+++ b/dao/cdrdao.drivers
@@ -104,6 +104,7 @@
 W|GENERIC|CRD-BP1600P|generic-mmc
 W|GENERIC|CRD-R800S|generic-mmc
 W|GENERIC|CRD-RW2|generic-mmc
+W|HL-DT-ST|DVDRAM GSA-H42L|generic-mmc|OPT_MMC_CD_TEXT
 W|HL-DT-ST|RW/DVD GCC-4120B|generic-mmc|OPT_MMC_CD_TEXT
 W|HP|9510i|generic-mmc|OPT_MMC_CD_TEXT
 W|HP|CD-Writer+ 7570|generic-mmc|OPT_MMC_CD_TEXT
